<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientsDialog</class>
 <widget class="QDialog" name="ClientsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Клиенты</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="filterLabel">
       <property name="text">
        <string>Фильтр</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterLineEdit">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="newclientPushButton">
       <property name="statusTip">
        <string>Создать клиента</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="kcrm.qrc">
         <normaloff>:/images/DocumentAdd.png</normaloff>:/images/DocumentAdd.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editPushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="kcrm.qrc">
         <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="clientsTableView">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Выбрать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="kcrm.qrc"/>
 </resources>
 <connections/>
</ui>
